1.0/tools/optimize-for-production.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: default
+type: guide
+shortname: Docs
+title: Optimize for production
+subtitle: Developer guide
+---
+
+{% include toc.html %}
+
+Reducing network requests is important for a performant app experience. In the {{site.project_title}} world, [Vulcanize](https://github.com/Polymer/vulcanize) is the name given to a build tool that lets you **concatenate** a set of elements and their HTML imported dependencies into a single file. Vulcanize recursively pulls in all your imports, flattens their dependencies and spits out something that can **reduce the number of network requests** your app makes.
+
+**Note:** for more great info on performance considerations worth keeping in mind when using HTML Imports, see [HTML Imports - #include for the web](http://www.html5rocks.com/en/tutorials/webcomponents/imports/#performance)
+{: .alert .alert-info }
+
+## Installation
+
+Install Vulcanize and its dependencies using [NPM](http://npmjs.org):
+
+    npm install -g vulcanize
+
+We recommend installing Vulcanize globally so you can run it from anywhere on the command-line. To install locally, simply omit the `-g` flag.
+
+## Usage
+
+Vulcanize can be used standalone from the command line, or as part of a gulp/grunt build chain.
+
+### Use vulcanize from the command line
+
+Assuming an input HTML file, elements.html, using a number of HTML imports, we can run it through Vulcanize by passing it as an argument as follows:
+
+    vulcanize elements.html
+
```

Comment:
This really doesn't seem like anything you'd want to do (unless you're redirecting output). Folks are likely to type this before they read the comment below, which should serve as a warning.

I'd be inclined to show the second command first, and mention the other form in passing ("If you omit the `-o elements.vulcanized.html`, Vulcanize prints the output to the terminal.").